module/enrichers.mjs

```diff
@@ -967,11 +1045,45 @@ async function rollAction(event) {
     try {
       switch ( type ) {
         case "damage": return await rollDamage(event);
+        case "item":
+          let item;
+	          // If UUID is provided, always roll that item directly
+	          if ( target.dataset.rollItemUuid ) item = await fromUuid(target.dataset.rollItemUuid);
+	          else if ( target.dataset.rollItemName ) {
+	            const actor = target.dataset.rollItemActor ? await fromUuid(target.dataset.rollItemActor) : null;
+	
+	            // If no actor is specified or player isn't owner, fall back to the macro rolling logic
+	            if ( !actor?.isOwner ) return rollItem(target.dataset.rollItemName);
+	
+	            // If a token is controlled and it has an item with the correct name, activate it
+	            if ( canvas.tokens.controlled[0]?.actor.items.getName(target.dataset.rollItemName) ) {
+	              item = canvas.tokens.controlled[0]?.actor.items.getName(target.dataset.rollItemName);
+	            }
+	
+	            // Otherwise check the specified actor for the item
+	            else {
+	              item = actor.items.getName(target.dataset.rollItemName);
+	
+	              // Display a warning to indicate the item wasn't rolled from the controlled actor
+	              if ( item && (canvas.tokens.controlled.length > 0) ) ui.notifications.warn(
+	                game.i18n.format("MACRO.5eMissingTargetWarn", {
```

Comment:
Looks like this warning doesnt get the type or name passed to it
![image](https://github.com/foundryvtt/dnd5e/assets/86370342/f124fe38-2cdb-49f3-94f2-dba3567cb5fc)

Reply:
Oh yeah, you'll want to change the `item` key to `name`, and pass in the localized "Item" for `type`.

Reply:
Since this was an existing warning, should I add a new one to `EDITOR.DND5E.Inline` instead?